src/frontends/paddle/src/op/less_equal.cpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "elementwise_ops.hpp"
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs less_equal(const NodeContext& node) {
+    return elementwise_ops<default_opset::LessEqual>(node);
+}
```

Comment:
it seems better to move this implement to[ 'elementwise_ops' cpp file](https://github.com/openvinotoolkit/openvino/blob/13a8d705b48d6f51c6e4a9b86da67cd42c59a2cb/src/frontends/paddle/src/op/elementwise_ops.cpp#L56)